<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateUser</class>
 <widget class="QDialog" name="updateUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="userUpdateTitle">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Update a user</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>70</y>
     <width>601</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="username">
        <property name="text">
         <string>UserName:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="usernameInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="admincheck">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Admin</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>First Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="firstNameinput">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Last Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lastNameInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="emailInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Total Spent:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="totalSpent">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="userSearch">
        <property name="text">
         <string>User Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateUserButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Update User</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>userSearch</sender>
   <signal>clicked()</signal>
   <receiver>updateUser</receiver>
   <slot>enableButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userSearch</sender>
   <signal>clicked()</signal>
   <receiver>updateUser</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateUserButton</sender>
   <signal>clicked()</signal>
   <receiver>updateUser</receiver>
   <slot>updatedUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updatedUser()</slot>
  <slot>search()</slot>
  <slot>enableButton()</slot>
 </slots>
</ui>
